server/views/pages/overview/partials/workAndInterestsTab/_inductionLongQuestionSet.njk

```diff
@@ -130,7 +130,11 @@ questions are different.
           <dd class="govuk-summary-list__value">
             <ul class="govuk-list">
               {% for job in workAndInterests.data.workInterests.longQuestionSetAnswers.jobs %}
-                <li>{{ job.jobType | formatJobType }}</li>
+                {% if job.jobType === 'OTHER' %}
```

Comment:
The "other" logic and prefixing of `Other - ` should have been here, not where we had it previously (that was my bad in an earlier PR)